.github/workflows/restore-test.sh

```diff
@@ -1,13 +1,31 @@
 #!/bin/bash
 
-set -e -u -o pipefail
+set -e -u -x -o pipefail
```

Comment:
Did you want to leave the -x in?  Same comment for test.sh

Reply:
Yeah, I was a little surprised they weren't in already, because it's always helpful for debugging CI failures.  I feel like the cost is low and the value is high for having the additional debug insight.